packages/fx-core/src/component/driver/devTool/installDriver.ts

```diff
@@ -249,7 +249,14 @@ export class ToolsInstallDriverImpl {
     this.context.addTelemetryProperties({
       [TelemetryProperties.driverArgs]: JSON.stringify({
         devCert: args.devCert,
-        func: args.func,
+        func: {
+          version: args.func?.version,
+          symlinkDir: args.func?.symlinkDir
+            ? args.func.symlinkDir === "./devTools/func"
```

Comment:
Do we treat exactly the same string as default, or the same folder as default? E.g., `devTools/func`, `devTools/func/`, `./devTools/func/` ...

Reply:
Good idea, updated.